client/src/components/Workbench/WelcomeMessages.js

```diff
@@ -32,14 +35,46 @@ const WelcomeMessages = () => {
         </Alert>
       </Route>
       <Route path={`${path}${ROUTES.DATA}`}>
-        <Alert
-          isOpen={welcomeMessages.datafiles}
-          toggle={() => onDismissWelcome('datafiles')}
-          color="secondary"
-          className="welcomeMessage"
-        >
-          This page allows you to upload and manage your files.
-        </Alert>
+        <div className="welcomeMessage">
+          <Alert
+            isOpen={welcomeMessages.datafiles}
+            toggle={() => onDismissWelcome('datafiles')}
+            color="secondary"
+          >
+            This page allows you to upload and manage your files.
+          </Alert>
+          {showWork2Message && (
+            <Alert
+              isOpen={welcomeMessages.work2}
+              toggle={() => onDismissWelcome('work2')}
+              color="warning"
+            >
+              <>
+                Notice: The Stockyard <code>/work</code> filesystem is being
+                deprecated. A new filesystem <code>/work2</code> is now
+                available and will eventually replace <code>/work</code>. During
+                the transition period, users must migrate any current data they
+                wish to retain. You can read more information about this change
+                in the{' '}
```

Comment:
[I have requested Design to review the text.](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1617313042019100)

Reply:
```suggestion
                Notice: The Stockyard <code>/work</code> filesystem is being
                deprecated. A new filesystem <code>/work</code> is now
                available and will eventually replace <code>/work</code>. During
                the transition period, migrate any data you wish to retain. Read
                more information about this change in the{' '}
```

— [source](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1617314437020900?thread_ts=1617313042.019100&cid=CQUA4D5KJ)

Reply:
Also part of [confirmed Design request](https://tacc-team.slack.com/archives/CQUA4D5KJ/p1617314687021700?thread_ts=1617313042.019100&cid=CQUA4D5KJ) is to use bold text. This bold should be done via CSS, not markup (because it is a stylistic choice, not a change in emphasis of text). *__How?__ In `<Message>` component CSS module stylesheet, ensure all `<code>` is rendered with `font-weight: bold`.*

Reply:
>  A new filesystem <code>/work</code> is now..

Any reason this is changed from /work2 to /work in your suggestion?

Reply:
> In <Message> component CSS module stylesheet, ensure all <code> is rendered with font-weight: bold
Since this isn't using Message, should I put it in a global stylesheet for the workbench or make a new css module for WelcomeMessage?

Reply:
The change from `/work2` to `/work` must be a typo from Craig. Fixed suggestion.

Oh, sorry. I mean't whatever CSS module it has or can have. Yeah, so WelcomeMesage's.